airbyte-ci/connectors/pipelines/pyproject.toml

```diff
@@ -9,7 +9,7 @@ description = "Packaged maintained by the connector operations team to perform C
 authors = ["Airbyte <contact@airbyte.io>"]
 
 [tool.poetry.dependencies]
-python = "^3.10"
+python = "~3.9" # This is pinned to 3.10 as currently there is an issue when deploying via dagster-cloud where a dependency does not have a prebuild wheel file
```

Comment:
```suggestion
python = "~3.9" # This is pinned to 3.9 as currently there is an issue when deploying via dagster-cloud where a dependency does not have a prebuild wheel file
```

Reply:
Realized I pinned the wrong pyproject here. Setting orchestrator/pyproject.toml python to ^3.9 should be sufficient to solve this